### DISCUSSION TITLE
new ToonSquid app icon

### DISCUSSION THREAD

**Discussion by Nelford** [keiwando/toonsquid#564] [Ideas]
I love Toonsquid, and it is so much better than the overhyped Procreate Dreams, but I feel like the cartoony Icon is holding people back from picking it up. I appreciate the irony considering this is an animation app! Also understand it comes from the fun opening animation.

However, for greater adoption I suggest something like this.

Disclaimer: I give full rights related to any inspiration to Toonsquid. Just want to be a helpful citizen :)
![Toonsquid Logo](https://github.com/keiwando/toonsquid/assets/166099261/2a0b2082-678c-4b2a-8d19-1f58640f71a9)

**Comment by DinoAwakening** [comment]
Well, IDK… The current icon is just so unique, and the same argument you could say from Gimpy, Moho or Firealpaca. Their logo are cartoonish, but it doesn’t undervalue those software, they’re known for what they’re used for. ToonSquid might not be that recognized, but eventually it will. Also, people might think it’s an Adobe App with that design, so probably no. Hope this helps :D

**Reply by Nelford** [reply]
Fair enough re: the Adobe app. I replaced the letters with the original eye. Although I have got to say, I have heard of Adobe, but none of the others you mentioned. I like the current logo, it is fun, but I was actively put off thinking Toonsquid was an amateur app from the logo, until I got frustrated with Procreate Dreams and looked for alternatives.

**Reply by budaloops** [reply]
ToonSquid ranks #3 in the Top Paid Apps in the ‘Graphics & Design’ category on the App Store (in Florida, USA). It may not be recognized by some, but it’s used by many
